app/components/search/SearchResults/SearchResults.tsx

```diff
@@ -75,38 +80,40 @@ const SearchResults = ({
           <NoResultsDisplay />
         ) : (
           <>
-            {searchQuery && (
-              <div className={styles.searchResultsHeader}>
-                <h2>{`${searchResults.nbHits} search results ${
-                  searchQuery && ` for ${searchQuery}`
-                }`}</h2>
-                <ClearSearchButton />
+            {searchResultsHeader}
+            {searchMapHitData.hits.map((hit: TransformedSearchHit, index) => (
+              <SearchResult
+                hit={hit}
+                key={`${hit.id} - ${hit.name}`}
+                ref={index === 0 ? handleFirstResultFocus : null}
+              />
+            ))}
+            <div
+              className={`${styles.paginationContainer} ${
+                hasNoResults ? styles.hidePagination : ""
+              }`}
+            >
+              <div className={styles.resultsPagination}>
+                {/* <Pagination
+                  padding={2}
+                  showLast={false}
+                  showFirst={false}
+                  translations={{
+                    previousPageItemText: "Prev",
+                    nextPageItemText: "Next",
+                  }}
+                /> */}
               </div>
-            )}
-            {searchMapHitData.hits.map(
-              (hit: TransformedSearchHit, index: any) => (
-                <SearchResult
-                  hit={hit}
-                  key={`${hit.id} - ${hit.name}`}
-                  ref={index === 0 ? handleFirstResultFocus : null}
-                />
-              )
-            )}
-            <ResultsPagination noResults={hasNoResults} />
+            </div>
           </>
         )}
       </div>
       <SearchMap
         hits={searchMapHitData.hits}
-        mapObject={googleMapObject}
-        setMapObject={setMapObject}
-        setAroundLatLng={setAroundLatLng}
```

Comment:
Overall my goal was to reduce prop bloat when that data could be fetched/set from the component itself.